lib/dev/ui/spinner.rb

```diff
@@ -40,6 +40,7 @@ def initialize(title, &block)
               end
             end
 
+            @force_full_render = false
```

Comment:
I'm not sure I like the use of `force_full_render`, Is there a way we can accomplish this without it? Can we just `render` whenever title changes?